DC/6.1.0/languages.dcl

```diff
@@ -972,29 +1214,36 @@ resource language vietnamese {
     locale_id = 1066
     enabled = false
     is_default_language = false
+    azure_language_code = 'vi'
+    elastic_language_code = ''
 }
 
 resource language xhosa {
     name = 'Xhosa'
     short_name = 'xh'
-    locale_id = 0
+    locale_id = 1076
     enabled = false
     is_default_language = false
+    azure_language_code = 'xh'
+    elastic_language_code = ''
 }
 
 resource language yiddish {
     name = 'Yiddish'
     short_name = 'ji'
-    locale_id = 0
+    locale_id = 1085
     enabled = false
     is_default_language = false
+    azure_language_code = ''
+    elastic_language_code = ''
 }
 
 resource language zulu {
     name = 'Zulu'
     short_name = 'zu'
-    locale_id = 0
+    locale_id = 1077
```

Comment:
I assume these `local_id`s have also ben fixed in the database?

Reply:
Yes, I have copied them from the update data script